docs-v1/run-random-beacon.adoc

```diff
@@ -280,15 +280,15 @@ This is a sample run command for illustration purposes only.
 
 [source,bash]
 ----
-export KEEP_CLIENT_ETHEREUM_PASSWORD=$(cat .secrets/eth-account-password.txt)
+export KEEP_ETHEREUM_PASSWORD=$(cat .secrets/eth-account-password.txt)
 export KEEP_CLIENT_CONFIG_DIR=$(pwd)/config
 export KEEP_CLIENT_PERSISTENCE_DIR=$(pwd)/persistence
 
 docker run -d \
 --entrypoint /usr/local/bin/keep-client
 --volume $KEEP_CLIENT_PERSISTENCE_DIR:/mnt/keep-client/persistence \
 --volume $KEEP_CLIENT_CONFIG_DIR:/mnt/keep-client/config \
---env KEEP_ETHEREUM_PASSWORD=$KEEP_CLIENT_ETHEREUM_PASSWORD \
+--env KEEP_ETHEREUM_PASSWORD=$KEEP_ETHEREUM_PASSWORD \
```

Comment:
This doc is for v1 and I believe `KEEP_CLIENT_ETHEREUM_PASSWORD` was the right choice there. Should we change it?

Reply:
52a93cfa893c83ce2b2cb845a7901edad6fe8bc1